ui/src/app/views/workspace-share/component.ts

```diff
@@ -175,9 +175,11 @@ export class WorkspaceShareComponent implements OnInit {
       this.autocompleteNoResults = true;
       return;
     }
-    this.userService.user(this.searchTerm).subscribe((response) => {
-      this.autocompleteLoading = false;
+    const searchTerm = this.searchTerm;
+    this.userService.user(searchTerm).subscribe((response) => {
+      if (this.searchTerm !== searchTerm) { return; }
```

Comment:
opt_nit: I'd suggest either dropping the brackets or making this multiline. Single line with brackets is a bit cluttered in my opinion.

Reply:
Lint currently complains without the brackets.

Reply:
OK - multiline then?

Reply:
👍